EVM and High Level Languages.md

```diff
@@ -0,0 +1,95 @@
+* Control structures
+* More than one variable can pass in and out of a function 
+* User ability to define custom functions 
+* Inheritance
+* Polymorphism 
+* Sand boxing 
+* Two-way interaction with other smart contracts 
+
+Simple transaction protocol initiates the deployment of very powerful applications on the EVM. The Ethereum protocol is trust-less; secured by peer-to-peer consensus and data encryption. It is very difficult, impossible, to access unauthorized information. The program code is verifiable and impossible to alter. Always available to every user as the same version and build. Pushing updates that users ignore can become a thing of the past.
+
+Prebuilt functions are built in the EVM.  Among other things these functions create data abstraction between the user and the actual blockchain implementation.  Direct features of the EVM are listed below that have an aggregate functionality of a self contained computer system.
+
+* Simple transaction methods for new smart contract deployment
+* Smart contract interaction only by address
+* Arguments and data pass in from the user and pass out to a user or contract
+* A stack type data structure holds local variables
+* It reads function call arguments and return-addresses
+* Logic functions can change the order in which instructions execute
+* Addressable persistent storage acts as a virtual hard drive 
+* Non-persistent storage acts as system memory
+
+A closer look at the EVM and dApp ecosystem show a complex orchestration of silently running programs and processes. Technology today offers many choices in a programming language presenting obstacles in compatibility. Cross-platform data abstraction requires a complex web of behind the scenes computing. 
+
+## Discrete Parts that make the EVM
+
+As stated before, the EVM is prebuilt with libraries of ready to use functions. Smart contract languages such as Solidity and Vyper are simple, feature-rich languages. Embedded function libraries manipulate data in ways making the EVM quasi-Turing complete. Turing complete system is so named on the basis that it can use a lot of math proving it can solve any algorithm.
+
+### EVM Opcodes and Bytecode
+
+<a href="https://imgur.com/J3lKgEy"><img src="https://i.imgur.com/J3lKgEy.png" title="source: imgur.com" /></a>
+
+Opcodes are a large library of functions preprogrammed into the EVM. A list of all Opcodes available is accessible [here](https://www.ethervm.io/#opcodes). Fewer resources handle these functions because they are small, 4-bit hex identifiers. Every function found in JavaScript or Python exists also in the EVM. Opcodes include functions for data manipulation, I/O, storage, security, and logic. High-level smart contract language compiles into bytecode. This is data passed into the EVM in hex that calls opcode functions and handles user arguments. The image located at the top of the page is a matrix of two digit hex opcodes. 
```

Comment:
```suggestion
Opcodes are a large library of functions preprogrammed into the EVM. You can find a list of all Opcodes [in the documentation](https://www.ethervm.io/#opcodes). Fewer resources handle these functions because they are small, 4-bit hex identifiers. Every function found in JavaScript or Python exists also in the EVM. Opcodes include functions for data manipulation, I/O, storage, security, and logic. High-level smart contract language compiles into bytecode. This is data passed into the EVM in hex that calls opcode functions and handles user arguments. The image located at the top of the page is a matrix of two digit hex opcodes. 
```